### ISSUE TITLE
Reverse open/close

### ISSUE THREAD

**Issue by nclsrm** [RainerStaude/hass-becker-component-plus-pybecker#27]
I've installed the integration and everything works perfectly.

However, I have a slight problem: I'm using the integration to control an awning, so the open and close states and up down functions are reversed. When I use the HA controls, it seems strange to "close" the awning to extend and "open" to retract.
Is there a way of inverting open/close? As I also use the controls on the HomeKit app, I suppose the changes would have to be made at the integration level and not just via an HA card with inverted controls.

**Comment by RainerStaude** [comment]
So far there is no option to invert the position in the Becker integration. 
You have two options:
1. Swap/correct up/down direction of your awning motor. There should be a switch on your motor.
2. Add a template cover similar to [here](https://community.home-assistant.io/t/cover-completely-reverse-invert/362662/60?u=rainers)
I'm wondering in case you have a remote. This should be inverted as well.

**Comment by domob1812** [comment]
I have a similar issue with one of my Becker blinds, where up and down is reversed.  I believe this was just a mistake of the contractor who installed it; but I can't easily reach the motor nor do I want to get a contractor again just to change this.  So while the remote is indeed reversed, too (and I'd prefer to fix that), at least it would be nice if I could have it proper inside HA.

It seems that the template cover is a good workaround and I'll probably do that.  But if you think it might be useful, I'm also happy to do a change to your integration that allows configuring to reverse the motor direction for each cover.  Is that something you think is a good idea and something you would like to accept into your code?

**Comment by RainerStaude** [comment]
There are two places, where this integration interacts with with Becker stuff.
1. Sending commands to the motor.
2. Receiving commands from the remote.
In my understanding, with reversed motor configuration, also the remote has to send reversed commands. 
Therefore one configuration parameter for reversed operation for both, sending and receiving should be sufficient.

**Comment by domob1812** [comment]
Yes, that is my understanding, too.  Do you think such a parameter would be useful and would accept a PR that adds it?  I can also understand the point of view that this is just additional "clutter" without any practical value, because a) the motor "should" be configured correctly in the first place, and b) even if it is not, one can work around it with a template cover (like I have done myself so far).
